extensions/amp-story/1.0/amp-story.js

```diff
@@ -417,6 +417,7 @@ export class AmpStory extends AMP.BaseElement {
     this.initializeListeners_();
     this.initializeListenersForDev_();
     this.initializePageIds_();
+    this.initializeInteractives_();
```

Comment:
nit for extra clarity (there's a lot of interactive stuff in a story):

```suggestion
    this.initializeInteractiveComponents_();
```